app/controllers/v0/health_records_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module V0
+  class HealthRecordsController < BBController
+    def refresh
+      resource = client.get_extract_status
```

Comment:
Excellent question. I will do my best to explain here.

MHV only makes the request to refresh the data once per day, subsequent requests to this endpoint only return the changes once the batched job they run has returned. For this reason I called this endpoint on our side just "refresh" and I propose we use this ultimately only on the backend. This would cache a result within Redis, and then subsequent requests to another endpoint, invoked by the Front-end could be used for polling to see when the data has been updated.

It's a slightly wonki process and I'm still trying to work through the details of this. I would ultimately like it if refresh returned an approved state, and set a redis store object with an id. And the front-end could then poll another endpoint to determine when the data is fresh. It's complicated.

For now, I'm just building it very simply as MHV has intended. We will most likely revisit and improve on this endpoint as we begin to implement the API.

But to answer your question, it should *not* be problematic.